src/components/ListingCard/ListingCard.js

```diff
@@ -152,7 +152,7 @@ ListingCardComponent.propTypes = {
   className: string,
   rootClassName: string,
   intl: intlShape.isRequired,
-  listing: propTypes.listing.isRequired,
+  listing: oneOfType([propTypes.listing, propTypes.ownListing]).isRequired,
```

Comment:
ListingCard is used on ProfilePage, and if we want to fetch own listings to show on that page, we need to use the ownListings endpoint which yields ownListings resources.